README.md

```diff
@@ -4,11 +4,40 @@ In this repository, we collect example scripts that can be useful for the team a
 
 ## Prerequisites
 
-TBD
+- install a python 3 interpreter (if not installed already)
+- create python virtual environment: `python3 -m venv venv`
+- activate the virtual environment: `source venv/bin/activate`
+- install dependencies: `pip install -q -r requirements.txt`
+
+### Authorization
+
+In order to authorize the requests:
+
+- create a `.config` file in the root directory
+- add your credentials in a `JSON` format into the `.config` file as following:
+
+```json
+{
+  "access_token": "${ACCESS_TOKEN}",
+  "client_id": "${CLIENT_ID}",
+  "client_secret": "${CLIENT_SECRET}",
+  "refresh_token": "${REFRESH_TOKEN}"
+}
+```
+
+Note: You need to authorize yourself through OAuth to get `access_token` and `refresh_token`. if token gets expired, the script will handle fetching a new one. here is how to use [OAuth](https://community.canvaslms.com/t5/The-Product-Blog/Connecting-Studio-OAuth-via-Postman/ba-p/259739)
+
```

Comment:
Once installed and configured, the user can test it by running:
```
bin/run examples/test --subdomain <school-subdomain>
```

For this we need an examples/test/main.py like this:
```
import argparse

from utils.utils import PublicAPIClient


def main():
    parser = argparse.ArgumentParser()
    parser.add_argument("--subdomain", type=str, required=True)
    args = parser.parse_args()

    public_api_client = PublicAPIClient(args.subdomain)
    public_api_client.refresh_tokens()
    response = public_api_client.request("get", "ping")

    if response.status_code == 200 and response.text == "pong":
        print("You have successfully configured your public API access.")
    else:
        print("Something is not okay")
        print("status_code:", response.status_code)
        print("response body:", response.text)


if __name__ == "__main__":
    main()
```